packages/neon/neon/lib/src/app.dart

```diff
@@ -272,8 +273,16 @@ class _NeonAppState extends State<NeonApp> with WidgetsBindingObserver, tray.Tra
                   builder: (final context, final capabilitiesSnapshot) {
                     final nextcloudTheme = capabilitiesSnapshot.data?.capabilities.theming;
                     return MaterialApp.router(
-                      localizationsDelegates: AppLocalizations.localizationsDelegates,
-                      supportedLocales: AppLocalizations.supportedLocales,
+                      localizationsDelegates: [
+                        ..._appImplementations.map((final app) => app.localizationsDelegate),
+                        ...AppLocalizations.localizationsDelegates,
+                      ],
+                      supportedLocales: {
+                        ..._appImplementations
+                            .map((final app) => app.supportedLocales)
+                            .reduce((final value, final element) => [...value, ...element]),
+                        ...AppLocalizations.supportedLocales,
+                      }.toList(),
```

Comment:
Set being a collection it's already iterable.

We can remove the `toList()` here.

Reply:
I added it because the analyzer was complaining about something but it's gone now :thinking: